cmd/cmd.go

```diff
@@ -914,8 +908,25 @@ func checkServerHeartbeat(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+func versionHandler(cmd *cobra.Command, _ []string) {
+	fmt.Printf("ollama version %s\n", version.Version)
+
+	client, err := api.ClientFromEnvironment()
+	if err != nil {
+		return
+	}
+
+	serverVersion, err := client.Version(cmd.Context())
+	if err != nil {
+		return
+	}
+
+	fmt.Printf("server version %s\n", serverVersion)
```

Comment:
Could we only print this if a remote host is specified with `OLLAMA_HOST`? and then something like:

```suggestion
        hostVersion, err := client.Version(cmd.Context())
	if err != nil {
		return
	}

	fmt.Printf("host version %s\n", serverVersion)
```